src/host/ut_host/TextBufferTests.cpp

```diff
@@ -1859,13 +1859,13 @@ void TextBufferTests::ResizeTraditionalRotationPreservesHighUnicode()
     // Fill it up with a sequence that will have to hit the high unicode storage.
     // This is the negative squared latin capital letter B emoji: 🅱
     // It's encoded in UTF-16, as needed by the buffer.
-    const auto bbutton = L"\xD83C\xDD71";
-    position = bbutton;
+    const auto bButton = L"\xD83C\xDD71";
+    position = bButton;

```

Comment:
Other variables in this file use camelCase (such as `readBack`).

Reply:
I think the author was trying to avoid it looking like the hungarian notation for "boolean Button" as `bButton`. But it's a test, so I don't mind either way.